### ISSUE TITLE
[Question] Poll mode with --load-model 

### ISSUE THREAD

**Issue by vvmatorin** [triton-inference-server/server#3575]
**Is your feature request related to a problem? Please describe.**
I have a Kubernetes cluster that I am deploying triton server on, but for each instance in that cluster I need to load specific models from the shared model-repository (not all of them), to balance the GPU load with the demand.

**Describe the solution you'd like**
I want an easy way to ignore models from the shared model repository when running the server with --model-control-mode=poll. Ideal way would be to use the --load-model option when starting the server and then only polling version changes over the models specified (ignoring the addition of the models to the repository).

Another way would be to use some kind of a flag to disable/ignore model in the config (e.g. setting instance count to 0).

**Describe alternatives you've considered**
Currently the best way to do that would be to run triton in manual mode with --load-model flag and manage all of the models manually. But that would also require me to implement the polling and model management functionality for new versions myself, which is a big overhead.

The second best way would be to have distinct model repositories for each instance, but that would incure overhead in storage and management, as I would have same models replicated over multiple repositories.

**Comment by CoderHam** [comment]
@vvmatorin correct me if I understand but you want to specify the models to load during server launch and have a mode where _only the loaded models will be polled_ for updates instead of the entire model repository?

**Comment by vvmatorin** [comment]
@CoderHam yes, you understood correctly. I want to have an option to either:

1. specify the models to load during the server launch and then only poll those models for updates instead of the entire model repository;
2. or allow polling for the whole directory (like it is implemented now), but have an option in the config (e.g. count=0 or gpus=[]), that would prevent the models from being loaded, until the config is changed and they are polled again.

**Comment by Legion2** [comment]
@vvmatorin have you looked at ModelMesh (https://github.com/kserve/modelmesh-serving) it automates the distribution, routing and load balancing of models on a cluster of servers.
